### PR TITLE
Beacd 8 | NES - with EORI and CHIEF number

### PR DESCRIPTION
**Pull request by kurtlogan**

### CODE REVIEW

File: app/utils/NextPage.scala [hunk 1]
```diff
@@ -33,13 +41,24 @@ trait NextPage[A, B] {
 
 object NextPage {
 
+  implicit val doYouHaveCHIEFRole: NextPage[DoYouHaveCHIEFRoleId.type,
+    DoYouHaveCHIEFRole] = {
+    new NextPage[DoYouHaveCHIEFRoleId.type, DoYouHaveCHIEFRole] {
+      override def get(b: DoYouHaveCHIEFRole)(implicit urlHelper: UrlHelper): Call =
+        b match {
+          case DoYouHaveCHIEFRole.Yes => Call("GET", urlHelper.emacEnrollmentsUrl(Enrolments.NewExportSystem))
+          case DoYouHaveCHIEFRole.No => ???
```
Comment (darbyo) [inline app/utils/NextPage.scala]: should have implementation?

File: app/utils/NextPage.scala [hunk 2]
```diff
@@ -61,18 +80,29 @@ object NextPage {
...
+    }
+  }
+
+  implicit val nesEori: NextPage[DoYouHaveEORINumberId.NES.type,
+    DoYouHaveEORINumber] = {
+    new NextPage[DoYouHaveEORINumberId.NES.type, DoYouHaveEORINumber] {
+      override def get(b: DoYouHaveEORINumber)(implicit urlHelper: UrlHelper): Call =
+        b match {
+          case DoYouHaveEORINumber.Yes => nesRoutes.DoYouHaveCHIEFRoleController.onPageLoad()
+          case DoYouHaveEORINumber.No => ???
```
Comment (darbyo) [inline app/utils/NextPage.scala]: as above?

File: app/utils/NextPage.scala (hunk 1 above)
Comment (kurtlogan) [reply app/utils/NextPage.scala]: Nope

File: app/utils/NextPage.scala (hunk 2 above)
Comment (kurtlogan) [reply app/utils/NextPage.scala]: Nope

**Pull request merged**